starky/src/permutation.rs

```diff
@@ -246,19 +246,23 @@ pub(crate) fn get_permutation_batches<'a, T: Copy>(
         .collect()
 }
 
-// TODO: Use slices.
```

Comment:
(These need to be copied now since `get_lde_values_packed` does a transpose, so I don't think slices would improve performance.)